.github/workflows/testing.yaml

```
@@ -0,0 +1,66 @@
+name: testing
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+
+  run-tests:
+    if: (github.event_name == 'push') ||
+        (github.event_name == 'pull_request' &&
+          github.event.pull_request.head.repo.full_name != github.repository) ||
+        (github.event_name == 'workflow_dispatch')
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        golang: ['1.23', '1.24']
```

Let's test on the minimal supported version and on the maximum.
```suggestion
        golang: ['1.23', 'latest']
```

Unlike linters, nothing should break here on update. But if it breaks, it's better to know about it.

---

fixed